modal/_functions.py

```diff
@@ -818,8 +813,7 @@ async def _load(self: _Function, resolver: Resolver, existing_object_id: Optiona
                     task_idle_timeout_secs=container_idle_timeout or 0,
                     concurrency_limit=concurrency_limit or 0,
                     pty_info=pty_info,
-                    cloud_provider=cloud_provider,  # Deprecated at some point
-                    cloud_provider_str=cloud.upper() if cloud else "",  # Supersedes cloud_provider
+                    cloud_provider_str=cloud if cloud else "",
```

Comment:
Just double-checking that the server doesn't expect uppercase values here anymore?

Reply:
yeah we lower case it server side anyway